bndtools.manual/tutorial.md

```diff
@@ -22,10 +22,20 @@ Installing Bndtools
 
 Please refer to the [Installation Instructions](installation.html).
 
+Create a Workspace Project (cnf)
+================================
+
+First we need to create a a Workspace Project. It will be named "cnf".
+
+1. From the File menu, select **New -&gt; Other -&gt; Bndtools -&gt; Bnd OSGi Workspace**.
```

Comment:
Is this still accurate for bndtools 3.3 builds? @seanbright just changed the menu location in a recent build.